flow-typed/forms-apps.js

```diff
@@ -100,7 +102,35 @@ declare type NewFormsListFormsApp = _NewFormsApp & {
   styles: FormsListStyles,
 }
 
-declare type NewFormsApp = NewFormsListFormsApp | NewVolunteersFormsApp
+declare type NewTilesFormsApp = _NewFormsApp & {
+  type: 'TILES',
+  slug: string,
+  tiles: Array<FormTile | HrefTile | AppTile>,
+  styles: FormsListStyles,
+}
+
+declare type Tile = {
+  icon: string
+}
+
+declare type FormTile = Tile & {
+  type: 'FORM'
+  form: Form,
+}
+
+declare type HrefTile = Tile & {
+  type: 'HREF'
+  url: string
+}
+
+declare type AppTile = Tile & {
```

Comment:
Should this be 
```suggestion
declare type ContainerTile = Tile & {
```
with type being 
```
type: 'CONTAINER',
```
?

or has this already been decided on?

Reply:
Not decision was made, so went with tiles 🤷